Tests/YCoreUITests/Protocols/ImageAssetTests.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  ImageAssetTests.swift
+//  YCoreUI
+//
+//  Created by Dev Karan on 21/12/22.
+//  Copyright © 2022 Y Media Labs. All rights reserved.
+//
+
+import XCTest
+import YCoreUI
+
+final class ImageAssetTests: XCTestCase {
+    func test_bundle() {
+        XCTAssertEqual(Flags.bundle, .module)
+        XCTAssertEqual(Icons.bundle, .module)
+    }
+    
+    func test_namespace() {
+        XCTAssertNil(Flags.namespace)
+        XCTAssertEqual(Icons.namespace, "Icons")
+        XCTAssertNil(Missing.namespace)
+    }
+    
+    func test_fallbackImage() {
+        XCTAssertNotNil(Flags.fallbackImage)
+        XCTAssertNotNil(Icons.fallbackImage)
+        XCTAssertEqual(Missing.fallbackImage, UIImage(systemName: "x.squareroot"))
+    }
+    
+    func test_loadImageWithNameSpace() {
+        Icons.allCases.forEach {
+            XCTAssertNotNil($0.loadImage())
+        }
+    }
+    
+    func test_loadImageWithoutNameSpace() {
+        Flags.allCases.forEach {
+            XCTAssertNotNil($0.loadImage())
+        }
+    }
+    
+    func test_imageAsset_defaultValues() {
+        XCTAssertEqual(DefaultImageAssets.bundle, .main)
+        XCTAssertEqual(DefaultImageAssets.defaultCase.image.pngData(), DefaultImageAssets.fallbackImage.pngData())
```

Comment:
```suggestion
        XCTAssertEqual(DefaultImageAssets.bundle, .main)
        XCTAssertNil(DefaultImageAssets.namespace)
        XCTAssertEqual(DefaultImageAssets.defaultCase.image.pngData(), DefaultImageAssets.fallbackImage.pngData())
```
Might as well test `namespace` as well.